libafl_frida/src/asan_errors.rs

```diff
@@ -181,15 +257,52 @@ impl AsanErrors {
                         writeln!(output).unwrap();
                     }
                 }
+                #[cfg(target_arch = "aarch64")]
                 writeln!(output, "pc : 0x{:016x} ", error.pc).unwrap();
 
+                #[cfg(target_arch = "x86_64")]
+                for (reg, name) in ASAN_SAVE_REGISTER_NAME
```

Comment:
perhaps this should be `ASAN_SAVE_REGISTER_NAMES`?

Reply:
fixed